code/src/layouts/dashboard-tabs/DashboardTabs.scss

```diff
@@ -0,0 +1,44 @@
+@import "@design-system";
+
+.dashboard-tabs {
```

Comment:
Don't forget about spa prefix)
And what do you think about changing the range of tabs? 
I mean - Products, Orders, Users - according to priority for business and maiby frequency of use

Reply:
We use `spa` prefix only for components, prefixed with `App`. About rearranging, it is useful remark, i will do it

Reply:
Discussed. I reordered it like so: orders - products - users